app/src/main/res/values/colors.xml

```diff
@@ -54,4 +54,16 @@
     <color name="security_bar_low">#eab41c</color>
     <color name="security_bar_good">#09a862</color>
     <color name="security_bar_best">#10b3d4</color>
+
+    <!-- New Design colors -->
+    <color name="green_new">#09a862</color>
```

Comment:
Can we use the names from zeplin pls, makes it easier to figure out which color it is